spec/breadcrumbs/validator_spec.rb

```diff
@@ -0,0 +1,214 @@
+# encoding: utf-8
+require 'spec_helper'
+
+require 'bugsnag/breadcrumbs/breadcrumb'
+require 'bugsnag/breadcrumbs/validator'
+
+RSpec.describe Bugsnag::Breadcrumbs::Validator do
+  let(:automatic_breadcrumb_types) { Bugsnag::Breadcrumbs::VALID_BREADCRUMB_TYPES }
+  let(:auto) { false }
+  let(:name) { "Valid message" }
+  let(:type) { Bugsnag::Breadcrumbs::ERROR_BREADCRUMB_TYPE }
+  let(:meta_data) { {} }
+
+  describe "#validate" do
+    it "does not 'ignore!' a valid breadcrumb" do
+      config = instance_double("Configuration")
+      allow(config).to receive(:automatic_breadcrumb_types).and_return(automatic_breadcrumb_types)
+      validator = Bugsnag::Breadcrumbs::Validator.new(config)
+
+      breadcrumb = instance_double("Breadcrumb")
+
+      allow(breadcrumb).to receive(:meta_data=)
```

Comment:
You can probably make these a little bit more concise by placing the method/message stubs with the initialization:
```ruby
breadcrumb = instance_double(
  Bugsnag::Breadcrumbs::Breadcrumb,
  :meta_data= => meta_data,
  :auto => auto,
  :name => name,
  :type => type,
  :meta_data => meta_data,
)
```